kotlinx.fuzz.junit/src/main/kotlin/kotlinx/fuzz/junit/KotlinxFuzzJunitEngine.kt

```diff
@@ -102,6 +125,36 @@ class KotlinxFuzzJunitEngine : TestEngine {
         else -> TestExecutionResult.failed(finding)
     }
 
+    private fun createReproducer(className: String, methodName: String): CrashReproducerGenerator {
+        val testClass = Class.forName(className).kotlin
+        val testInstance = testClass.testInstance()
+        val testMethod = testClass.declaredMemberFunctions.single { it.name == methodName }
```

Comment:
This line looks very dangerous to me. Exception from here will prevent `finishExecution` which parses logs into csv. This will lead to plugin failure, which is very inconvenient.
Some ideas:
1. replace `single` with `singleOrNull { ... } :? return null`
2. `Koltinx.Fuzz.JunitEngine.Companion` has methods `isFuzzTarget` and `isJazzerTarget` which checks for annotation and parameter types. You can extract them and use here. Just be carefull, and avoid generating reproducers for jazzer targets